source/user-manual/capabilities/file-integrity/advanced-settings.rst

```diff
@@ -67,6 +67,33 @@ You need to install the audit daemon if you don’t have it already installed on
          # cp /usr/sbin/audisp-af_unix /sbin/audisp-af_unix
          # rc-service auditd restart
 
+.. note::
+
+   On certain systems, auditd is configured with a default rule that causes all audit rule processing to be skipped. This disables any whodata-related configuration of Wazuh.
+   
+   Link to auditd Linux manual: https://man7.org/linux/man-pages/man8/auditctl.8.html#DISABLED_BY_DEFAULT
+
+   Check auditd rules:
+  
+   .. code-block::
+
+      auditctl -l
+      -a never,task
+
+   You need to remove that rule, or include a specific filter for the syscheck daemon in the auditd rule file ``/etc/audit/rules.d/audit.rules``:
+  
+   .. code-block::
+
+      -a always,task -F exe=‘/var/ossec/bin/wazuh-syscheckd’
+      -a never,task
```

Comment:
if we are keeping this rule we need to emphasize the filter line with for example `:emphasize-lines: 1` in this code block

Reply:
Done